account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -126,47 +127,62 @@ def _default_sequence_ids(self):
     )
 
     def create_bank_journals(self):
-        AccountJournal = self.env["account.journal"].sudo()
-        AccountAccount = self.env["account.account"].sudo()
-        bank_journals = AccountJournal.search(
+        aj_model = self.env["account.journal"].sudo()
+        aa_model = self.env["account.account"].sudo()
+        rpb_model = self.env["res.partner.bank"].sudo()
```

Comment:
In normal python, `CapWords` style is used in class names. In Odoo, models are classes and saved in the registry. That's why it's a common practice to name a model variable with that style, when it represents the model without any record associated.

I'm not saying it's better or worse, but since it was like this before, I think it's good to keep it as it was, to keep the patch smaller.

Just a suggestion.